wercker.yml

```diff
@@ -41,6 +41,15 @@ build:
           bin/cfmt
           git diff --exit-code
     - script:
+        name: gas
+        code: |
+          go get github.com/GoASTScanner/gas
```

Comment:
Do we want to pin this to a particular revision of gas? Or maybe put it in the docker image? See the goimports stuff in `docker/wercker/Dockerfile` (and instructions in the Readme there). I can think of tradeoffs on both sides.